.NET/Microsoft.Recognizers.Definitions.Common/Korean/DateTimeDefinitions.cs

```diff
@@ -133,7 +134,7 @@ public static class DateTimeDefinitions
       public const string DurationHalfRegex = @"ㅂ";
       public const string DurationRelativeDurationUnitRegex = @"(지난|저번|작(?=년))";
       public const string DurationDuringRegex = @"(동안)";
-      public const string DurationSomeRegex = @"(몇|여러)";
```

Comment:
Why remove this in Korean?

Reply:
This is handled differently in Korean (as inexact number with unit) and having it extracted also here resulted in parsing conflicts.